src/main/kotlin/frc/robot/subsystems/swerve/SwerveModule.kt

```diff
@@ -43,6 +43,9 @@ class SwerveModule
     val position
         get() = inputs.absolutePosition
 
+    val velocity
+        get() = io.velocity
```

Comment:
This will not work in replay.

Reply:
0ce68bfa214a97fbaeec65b40e7a25e6c878c8e6